src/config.js

```diff
@@ -160,6 +189,11 @@ export function newConfig() {
 
       // default max nested iframes for referer detection
       maxNestedIframes: DEFAULT_MAX_NESTED_IFRAMES,
+
+      // default bidders iframes urls
+      userSync: {
+        topics: DEFAULT_IFRAMES_CONFIG
```

Comment:
actually i think we can just leave this out altogether; this would add configuration on an optional module regardless of it it were included. 

When I said move to config, I should have been more specific, I meant we would allow publishers to choose these with their configuration, not default to anyone.